src/Sulu/Component/Content/Compat/Structure/StructureBridge.php

```diff
@@ -124,7 +120,7 @@ public function getWebspaceKey()
      */
     public function getUuid()
     {
-        return $this->getDocument()->getUuid();
+        return $this->getDocument()->getUuid($this->document);
```

Comment:
Are you sure there is not just an incorrect type hint? I guess `$this->getDocument()` is already returning `$this->document`, so what's the point in passing it again?